shell/common/shell.h

```diff
@@ -90,6 +90,10 @@ class Shell final : public PlatformView::Delegate,
   Rasterizer::Screenshot Screenshot(Rasterizer::ScreenshotType type,
                                     bool base64_encode);
 
+  /// Schedules a frame to be rendered and waits for it to finish.
+  ///\returns true when there has been a timeout.
+  bool WaitForFrameRender(fml::TimeDelta timeout);
```

Comment:
Can we also specify which thread is this function supposed to be called? Platform thread?

Reply:
Done, specified which threads can't call it.